### PR TITLE
It's time to migrate to Laravel 6

### PR DESCRIPTION
**Pull request by alphamarket**
Pull request contains the configuration that supports `Laravel 6`, there was a problem issued in the test unit which got fixed too.

### CODE REVIEW

File: src/Postcode.php [hunk 1]
```diff
@@ -171,7 +171,7 @@ public function setCountry($country = null)
     public function mutatePostcode($postcode)
     {
         // Ensure the postcode is all upper case with no leading or trailing spaces
-        return strtoupper(trim($postcode));
+        return strtoupper(preg_replace('/[ \n\t]+/i', '', trim($postcode)));
```
Comment (Daursu) [inline src/Postcode.php]: why was this change necessary? This will remove the white spaces within the postcode as well which could cause side effects

Comment (alphamarket) [reply src/Postcode.php]: I know It replaces the white space... your test units would fail without it, If your test units are not up-to-date I can revert this change.

Comment (Daursu) [reply src/Postcode.php]: okay it looks good
